EventFiltering/Zorro.cxx

```diff
@@ -191,6 +253,9 @@ bool Zorro::isSelected(uint64_t bcGlobalId, uint64_t tolerance)
         mAnalysedTriggersOfInterest->Fill(i);
         mZorroSummary.increaseTOIcounter(mRunNumber, i);
       }
+      if (mToiHisto && lastSelectedIdx != mLastSelectedIdx) {
```

Comment:
Instead of having `mToiHisto` as a datamember, you can add an optional histogram pointer to `isSelected` and fill it only if it is provided

Reply:
Dear @mpuccio , Thanks for your comments. I agree with them and have submitted a PR to make the modifications.